base_layer/core/src/proof_of_work/monero_rx/helpers.rs

```diff
@@ -552,6 +552,7 @@ mod test {
         };
         let hash = block_header.merge_mining_hash();
         append_merge_mining_tag(&mut block, hash).unwrap();
+        #[allow(clippy::redundant_clone)]
         let mut block_header2 = block_header.clone();
```

Comment:
```suggestion
        let mut block_header2 = block_header;
```

Reply:
The clone and `#allow` is to satisfy `cargo clippy --all-targets --all-features -- -D warnings`